patches/kaggle_gcp.py

```diff
@@ -88,10 +88,10 @@ def __init__(self, parentCredential=None, quota_project_id=None):
 class _DataProxyConnection(Connection):
     """Custom Connection class used to proxy the BigQuery client to Kaggle's data proxy."""
 
-    API_BASE_URL = os.getenv("KAGGLE_DATA_PROXY_URL")
+    DEFAULT_API_ENDPOINT = os.getenv("KAGGLE_DATA_PROXY_URL")
```

Comment:
This can be removed.

Reply:
If anywhere else in the future code looks at Connection.DEFAULT_API_ENDPOINT, I think we want this set?

Reply:
I think it would be best to remove it for now because we set the `api_endpoint` field which means the default value won't be used. If future code looks for it, then the test would fail and we could fix. I have a preference to keep these hacks as simple as needed.